view/frontend/web/js/view/payment/method-renderer/adyen-paypal-method.js

```diff
@@ -32,14 +36,45 @@ define(
             },
             renderActionComponent: function(resultCode, action, component) {
                 fullScreenLoader.stopLoader();
-
+                   this.token = action.sdkData.token;
                 this.actionComponent = component.handleAction(action);
             },
             handleOnFailure: function(response, component) {
                 this.isPlaceOrderAllowed(true);
                 fullScreenLoader.stopLoader();
                 component.handleReject(response);
             },
+            handleOnError:  function (error, component) {
+                console.log("onError:",error);
```

Comment:
Can we safely log any error message coming from the component? What would you think about logging to console only if the environment is TEST ([example](https://github.com/Adyen/adyen-magento2/blob/024fe63d83e6d7360bee290540df1d86652ef738/view/frontend/web/js/model/adyen-checkout.js#L65))?

Reply:
Oh, left from my testing. Updating it. Thank You!